src/components/form/textInput/index.js

```diff
@@ -8,6 +8,7 @@ const TextInput = ({
   icon,
   type = "text",
   disabled,
+  readOnly,
```

Comment:
what is the purpose on readOnly if we already have disabled

Reply:
I kept getting a 'set ReadOnly error' so switched to readOnly instead of disabled. Will delete disabled.

Reply:
Let's keep disabled in place to avoid conflicts and in case it will be use to use in the future